browser/components/ContributionsTable.tsx

```diff
@@ -97,84 +93,50 @@ class ContributionsTable extends React.Component<Props, State> {
     });
   };
 
-  static contributorEntry = (item, exists) => {
-    if (exists) {
-      return (
-        <Column
-          key={item.name + '_contributor'}
-          header={<Cell>Contributor Alias</Cell>}
-          cell={
-            <TableTextCell
-              data={item}
-              field="contributor_alias"
-              col="contributor_alias"
-            />
-          }
-          width={125}
-        />
-      );
-    }
-  };
-
   static renderTables = (name, item, tableProps) => {
+    let count = 0;
     return (
-      <div key={name}>
-        <h4>{name}</h4>
-        <Table
-          key={item.name + '_table'}
-          rowsCount={item.length}
-          rowHeight={50}
-          headerHeight={50}
-          width={850}
-          maxHeight={500}
-        >
-          <Column
-            key={item.name + '_summary'}
-            header={<Cell>Summary</Cell>}
-            cell={
-              <TableSummaryCell data={item} field="contribution_description" />
-            }
-            width={tableProps.summaryWidth}
-          />
-          <Column
-            key={item.name + '_url'}
-            header={<Cell>Context URL</Cell>}
-            cell={<TableLinkCell data={item} field="contribution_url" />}
-            width={75}
-          />
-          <Column
-            key={item.name + '_submission_date'}
-            header={<Cell>Submission Date</Cell>}
-            cell={
-              <TableDateCell data={item} field="contribution_submission_date" />
-            }
-            width={100}
-          />
-          <Column
-            key={item.name + '_approval_status'}
-            header={<Cell>Approval Status</Cell>}
-            cell={
-              <TableApprovalStatusCell data={item} field="approval_status" />
-            }
-            width={100}
-          />
-          <Column
-            key={item.name + '_github_status'}
-            header={<Cell>Github Status</Cell>}
-            cell={
-              <TableGitHubStatusCell
-                data={item}
-                field="contribution_github_status"
-              />
-            }
-            width={100}
-          />
-          {tableProps.contributor ? (
-            ContributionsTable.contributorEntry(item, tableProps.contributor)
-          ) : (
-            <div />
-          )}
-        </Table>
+      <div key={count++}>
```

Comment:
This `count` will always be `1` in this function. Having the key be `name` should be fine; did that not work?

Reply:
yep, It should have worked fine. I have no idea how it got in there but i will fix it.